src/components/Details/Legislation/LegislationOverview.tsx

```diff
@@ -104,23 +104,27 @@ const LegislationOverview: FC<LegislationOverviewProps> = ({
         <div>
           <dt>Latest Meeting:</dt>
           <dd>
-            <Link to={`/events/${event.id}`}>
-              {event.event_datetime.toLocaleDateString("en-US", {
-                month: "short",
-                day: "numeric",
-                year: "numeric",
-              })}
-            </Link>
-          </dd>
-        </div>
-        <div>
-          <dt>Latest Document:</dt>
-          <dd>
-            <a target="_blank" rel="noopener noreferrer" href={document.url}>
-              {document.name}
-            </a>
+            {event && (
+              <Link to={`/events/${event.id}`}>
+                {event.event_datetime.toLocaleDateString("en-US", {
+                  month: "short",
+                  day: "numeric",
+                  year: "numeric",
+                })}
+              </Link>
+            )}
```

Comment:
Could pull the conditional rendering to the `div` level in line 104 so that `Latest Meeting` is also not displayed if there's no event?

Reply:
This diff looks confusing. Basically, make the conditional rendering for `Latest event` similar to the conditional rendering you currently have for `Latest Document`.